src/enrichers/legacy-duid.js

```diff
@@ -9,7 +9,7 @@ export function enrich (state, storageHandler) {
   console.log('legacy-duid.enrich', state)
   const duidLsKey = getLegacyIdentifierKey()
   try {
-    if (state.appId && storageHandler.hasLocalStorage()) {
+    if (state.appId && storageHandler.localStorageIsEnabled()) {
```

Comment:
Omg... :(

Reply:
I did this change to have the same behaviour for the check in prebid and builder.
Prebid's `hasLocalStorage` is returning `!!window.localStorage`, so it just checks that the localStorage exists in the page.
Whereas our implementation of `hasLocalStorage`: puts the value into the ls -> gets the value from ls -> checks that the value was successfully written. This is what `localStorageIsEnabled` does in prebid. 

So the name is changes in our codebase to be `localStorageIsEnabled`.

Reply:
Yes yes! Great find!